picard/ui/sortablecheckboxlist.py

```diff
@@ -101,6 +106,33 @@ def _emit_changed(self):
         if not self.__no_emit:
             self.changed.emit(self.__items)
 
+    def set_item_marking(self, row, value):
```

Comment:
Are there any plans to use that feature in the near future? If not, remove it, it's easy enough to add it again if needed. That would also allow merging set_all_marked and remove_marked into one function (looking at the documentation of a few qt widgets, this functions seems to be called `clear` most of the time).